src/renderer/components/RouterComponents/Description/Description.tsx

```diff
@@ -26,19 +15,12 @@ const Description = () => {
       <button onClick={() => dispatch(installGatsbyCLI())}>
         Install gatsby-cli
       </button>
-      <button
-        onClick={() =>
-          dispatch(displayUserData(currentUser.auth.accessToken.value))
-        }
-      >
-        Display user data
-      </button>
       <p>
-        Welcome <b>{currentUser.data.nick}</b>!
+        Welcome <b>{currentUser.data?.nick || ''}</b>!
```

Comment:
The correct syntax for optional chaining would be `currentUser.data?.nick ?? ''`

Reply:
Cool stuff, didn't know about nullish coalescing operator before. 
Either way this was just placeholder when working and isn't really necessary as data is already there when this screen is rendered and in the end will be removed so won't bother.